maro/cli/data_pipeline/data_center.py

```diff
@@ -0,0 +1,173 @@
+import gzip
```

Comment:
split the data pipeline and BE to two pr.

Reply:
@kyu-kuanwei

Reply:
okay